lexicons/app/bsky/moderation/defs.json

```diff
@@ -64,29 +46,22 @@
     },
     "modServicePolicies": {
       "type": "object",
-      "required": ["reportReasons", "labelValues"],
```

Comment:
I'm dropping the custom report reasons for now, we'll FF with it but I think we gotta eject for now